e2e/cypress/integration/enterprise/system_console/team_members_spec.js

```diff
@@ -0,0 +1,228 @@
+            // # Make the user an admin
+            cy.findByText('Make Team Admin').should('be.visible').click();
+        });
+
+        // # Search for user2
+        cy.get('#teamMembers .DataGrid_search input').clear().type(user2.email);
+        cy.wait(TIMEOUTS.TINY); // Timeout required to wait for timeout that happens when search input changes
+
+        // * Assert that the user is now added to the members block and contains text denoting that they are New
+        cy.get('#teamMembers .DataGrid_rows').children(0).should('contain', user2.email).and('contain', 'New');
+
+        // # Search for sysadmin
+        cy.get('#teamMembers .DataGrid_search input').clear().type(sysadmin.email);
+        cy.wait(TIMEOUTS.TINY); // Timeout required to wait for timeout that happens when search input changes
+
+        // * Assert that searching for users after adding users returns only relevant search results
+        cy.get('#teamMembers .DataGrid_rows').children(0).should('contain', sysadmin.email);
+
+        // # Attempt to save
+        saveConfig();
+
+        // # Click the team edit link
+        cy.findByTestId(`${team.display_name}edit`).click();
+
+        // # Search user1 that we know is now in the team again
+        cy.get('#teamMembers .DataGrid_search input').scrollIntoView().clear().type(user1.email);
+        cy.wait(TIMEOUTS.TINY); // Timeout required to wait for timeout that happens when search input changes
+        cy.get('#teamMembers .DataGrid_loading').should('not.be.visible');
+
+        // * Assert that the user is now saved as an admin
+        cy.get('#teamMembers .DataGrid_rows').children(0).should('contain', user1.email).and('not.contain', 'New').and('contain', 'Team Admin');
+
+        // # Open the user role dropdown menu
+        cy.get(`#userGridRoleDropdown_${user1.username}`).click();
+
+        // * Verify that the menu is opened
+        cy.get('.Menu__content').should('be.visible').within(() => {
+            // # Make the user a regular member again
+            cy.findByText('Make Team Member').should('be.visible').click();
+        });
+
+        // * Assert user1 is now back to being a regular member
+        cy.get('#teamMembers .DataGrid_rows').children(0).should('contain', user1.email).and('not.contain', 'New').and('contain', 'Team Member');
+
+        // # Search user2 that we know is now in the team again
+        cy.get('#teamMembers .DataGrid_search input').scrollIntoView().clear().type(user2.email);
+        cy.wait(TIMEOUTS.TINY); // Timeout required to wait for timeout that happens when search input changes
+        cy.get('#teamMembers .DataGrid_loading').should('not.be.visible');
+
+        // * Assert user2 is now saved as a regular member
+        cy.get('#teamMembers .DataGrid_rows').children(0).should('contain', user2.email).and('not.contain', 'New').and('contain', 'Team Member');
+
+        // # Attempt to save
+        saveConfig();
+    });
```

Comment:
Since there are other tests which are dependent on user-1 and user-2 to be part of the default team, can you either
a) Work on a new team (You can create the team at the start of the test, and add the members to them)
b) Add an after block and add the members user-1 and user-2 to the default team. 

This might be required both in channel_members_spec and team_members_spec

Reply:
I have modified the tests to create a new team (and channel) at the start and then delete the team and channel afterwards